### DISCUSSION TITLE
Thank you a lot!

### DISCUSSION THREAD

**Discussion by powerman** [mywalkb/LSPosed_mod#6] [General]
Looks like this LSPosed version is main and only way to continue conveniently using XPrivacyLua.
I've checked other ways (Tasker automations) but they are either outdated or looks unsafe (one which kills LSPosed for 1 second after editing it's database - I suppose this might result in bypassing XPrivacyLua restrictions within that 1 second).

**Comment by mywalkb** [comment]
I'm glad to read it.
In my mod there is also API, a module app can change its scope directly without root access.
I saw that now after more than a year they are doing it in the original project.

**Comment by SetOfAllSets** [comment]
+1, lsposed was causing my zygisk to crash and I knew that since it was unmaintained it wouldn't get fixed, after searching for a bit I found this. Thank you for this wonderful project.

**Reply by mywalkb** [reply]
Great, could you tell me which ROM are using?
My version support stripped libraries and android 15.

**Comment by bLind17** [comment]
I am on a LineageOS with Android 14, LSPosed did not open, shell kept crashing.
Huge thanks to you, all works fine again :)
Can I buy you a coffee?

**Reply by mywalkb** [reply]
I'm glad it works, for the coffee act as if I accepted thanks.
